Eat_and_Greet/src/comps/HeaderBackButton/index.styles.tsx

```diff
@@ -0,0 +1,21 @@
+import { StyleSheet } from "react-native";
+
+const localStyles = StyleSheet.create({
+  container: {
```

Comment:
Don't adore the naming here, but I'm not gonna block the release. In the future avoid just container as I'd like to know what's being contained or why it's special.